### PR TITLE
add a new language - Ukrainian && fix bug with dayjs locale

### PR DESCRIPTION
**Pull request by mikollaay**
**Type:**

The following has been addressed in the PR:

*  There is a related issue?
*  Unit or Functional tests are included in the PR

**Description:**

### CODE REVIEW

File: src/components/LanguageSwitch/LanguageSwitch.tsx [hunk 1]
```diff
@@ -139,10 +144,7 @@ const LanguageSwitch = () => {
                     .map(language => {
                       const { icon, translation } = getTranslatedCurrentLanguageDetails(t, language);
                       return (
-                        <StyledMenuItem
-                          key={language}
-                          onClick={handleSwitchLanguage(language)}
-                          selected={userLanguage === translation}>
+                        <StyledMenuItem key={language} onClick={handleSwitchLanguage(language)} selected={userLanguage === translation}>
```
Comment (juanpicado) [inline src/components/LanguageSwitch/LanguageSwitch.tsx:147]: It seems linter is yelling, here, could you check that?

Comment (mikollaay) [reply src/components/LanguageSwitch/LanguageSwitch.tsx]: fixed

File: partials/storage/jquery/package.json [hunk 2]
```diff
@@ -4981,7 +4981,7 @@
 			"shasum": "958ce29e81c9790f31be7792df5d4d95fc57fbca"
 		}
 	},
-	"_rev": "61-e6be890a78963127",
+	"_rev": "62-9bab21b06ddb36b3",
```
Comment (priscilawebdev) [inline partials/storage/jquery/package.json:4984]: could you please check this file? I believe you committed this by mistake.

File: src/components/Icon/img/ukraine.svg [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" id="ukraine">
```
Comment (priscilawebdev) [inline src/components/Icon/img/ukraine.svg:1]: Maybe you could use https://jakearchibald.github.io/svgomg/ to compress this SVG

**Comment by priscilawebdev** [comment]
@mikollaay thank you for your contribution 🙂Please could you describe the issue you fixed?

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -23,6 +23,7 @@ const Footer: React.FC = () => {
               <Flag name="china" size="md" />
               <Flag name="austria" size="md" />
               <Flag name="germany" size="md" />
+              <Flag name="ukraine" size="md" />
```
Comment (priscilawebdev) [inline src/components/Footer/Footer.tsx:26]: Sorry, but this should not be part of this PR

File: i18n/translations/es-ES.json [hunk 5]
```diff
@@ -143,7 +143,8 @@
         "spanish": "Español",
         "german": "Alemán",
         "chinese": "Chino",
-        "french": "francés"
+        "french": "Francés",
+        "ukraine": "Ucranio"
```
Comment (juanpicado) [inline i18n/translations/es-ES.json:147]: "ukraine": "Ucraniano"

File: partials/storage/jquery/package.json (hunk 2 above)
Comment (mikollaay) [reply partials/storage/jquery/package.json]: fixed

File: src/components/Icon/img/ukraine.svg (hunk 3 above)
Comment (mikollaay) [reply src/components/Icon/img/ukraine.svg]: fixed

File: i18n/translations/pt-BR.json [hunk 6]
```diff
@@ -143,7 +143,8 @@
         "spanish": "Espanhol",
         "german": "Alemão",
         "chinese": "Chinês",
-        "french": "Francês"
+        "french": "Francês",
+        "ukraine": "Ucraniano"
     },
     "help-to-translate": "Ajude a traduzir",
```
Comment (priscilawebdev) [inline i18n/translations/pt-BR.json:149]: Please could you also translate `"help-to-translate"` and `"change-language" `?

File: src/components/Footer/Footer.tsx (hunk 4 above)
Comment (juanpicado) [reply src/components/Footer/Footer.tsx:26]: Yes, as @priscilawebdev suggests. The footer flag is like a memorial place for those that settled the bases of this project. It's a reserved area and has to be earned with contributions during a long period of time (it's a non-written rule), it just happened. 

The current ones represent the countries where live / born the most active maintainers. 👍 . The latest one we added was Germany (my new home), looking forward to adding new flags.

Just wanna give you more context so don't get it wrong.

**Comment by juanpicado** [comment]
Bundlesize is failing, please ignore it, we should increase size since we are adding new translations, not a big problem :)

File: i18n/translations/pt-BR.json (hunk 6 above)
Comment (mikollaay) [reply i18n/translations/pt-BR.json:149]: what language? I added the Ukrainian language uk-UA.json

"help-to-translate": "Допоможіть перекласти",
"change-language": "Змінити мову"

File: i18n/translations/es-ES.json (hunk 5 above)
Comment (mikollaay) [reply i18n/translations/es-ES.json]: +

File: src/components/Footer/Footer.tsx (hunk 4 above)
Comment (mikollaay) [reply src/components/Footer/Footer.tsx]: ++

**Comment by mikollaay** [comment]
Знімок екрана 2020-05-02 о 23 24 59

Date is always in English

allocated fix code last commit 53f8d5611e2b0920059f6c6cb9e6962efb26911f

File: src/App/load-dayjs-locale.ts [hunk 7]
```diff
@@ -19,12 +19,13 @@ function loadDayJSLocale() {
   const fallbackLanguage = getFallFackLanguage();
   const locale = i18n.language || fallbackLanguage;
 
-  // dayjs loades en-US by default
-  if (!locale || locale === 'en-US') {
-    return;
-  }
-
   switch (locale.toLowerCase()) {
+    case 'en-us':
```
Comment (priscilawebdev) [inline src/App/load-dayjs-locale.ts:23]: In this case I believe that `en-us `should be the default

File: src/App/load-dayjs-locale.ts [hunk 8]
```diff
@@ -19,12 +19,13 @@ function loadDayJSLocale() {
   const fallbackLanguage = getFallFackLanguage();
   const locale = i18n.language || fallbackLanguage;
 
-  // dayjs loades en-US by default
-  if (!locale || locale === 'en-US') {
-    return;
-  }
-
   switch (locale.toLowerCase()) {
```
Comment (priscilawebdev) [inline src/App/load-dayjs-locale.ts:22]: `locale?.toLowerCase()`

File: src/design-tokens/ThemeProvider.tsx [hunk 9]
```diff
@@ -22,7 +24,8 @@ const ThemeProvider: React.FC = ({ children }) => {
 
   useEffect(() => {
     changeLanguage();
-  }, [language, changeLanguage]);
+    loadDayJSLocale();
```
Comment (priscilawebdev) [inline src/design-tokens/ThemeProvider.tsx:27]: now  I know the bug you have fixed... good catch! thanks :)

File: src/design-tokens/ThemeProvider.tsx [hunk 10]
```diff
@@ -3,6 +3,8 @@ import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
 import { ThemeProvider as EmotionThemeProvider } from 'emotion-theming';
 import i18next from 'i18next';
 
+import loadDayJSLocale from '../App/load-dayjs-locale';
```
Comment (priscilawebdev) [inline src/design-tokens/ThemeProvider.tsx:6]: maybe now it would be better if we move `load-dayjs-locale `file to design-tokens 😉

**Comment by priscilawebdev** [comment]
@mikollaay  if you don't mind I will update with a small fix, so then the tests will pass and we can have it merged ;-)

**Comment by mikollaay** [comment]
👍
